.github/workflows/cherryPick.yml

```diff
@@ -69,6 +69,23 @@ jobs:
           ref: staging
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Decrypt Botify GPG key
+        if: github.actor == 'OSBotify'
+        run: cd .github/workflows && gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
+        env:
+          LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+
+      - name: Import Botify GPG Key
+        if: github.actor == 'OSBotify'
+        run: cd .github/workflows && gpg --import OSBotify-private-key.asc
+
+      - name: Set up git for Botify
+        if: github.actor == 'OSBotify'
+        run: |
+          git config user.signingkey DBF63700F60F5530
+          git config commit.gpgsign true
+          git config user.email infra+osbotify@expensify.com
```

Comment:
```suggestion
          git config user.name OSBotify
          git config user.email infra+osbotify@expensify.com
```

Reply:
`user.name` is set below but we can set it here to be explicit